lib/micro/case.rb

```diff
@@ -5,14 +5,14 @@
 
 module Micro
   class Case
-    require 'micro/case/version'
-    require 'micro/case/result'
-    require 'micro/case/error'
-    require 'micro/case/safe'
-    require 'micro/case/strict'
-    require 'micro/case/flow/reducer'
-    require 'micro/case/flow'
-    require 'micro/case/safe/flow'
+    require_relative 'case/version'
+    require_relative 'case/result'
+    require_relative 'case/error'
+    require_relative 'case/safe'
+    require_relative 'case/strict'
+    require_relative 'case/flow/reducer'
+    require_relative 'case/flow'
+    require_relative 'case/safe/flow'
```

Comment:
@linqueta, thanks for your contribution. But, I think that is an unnecessary change because the RubyGems resolves the gems' paths, and I don't see this kind of approach in other ones (.e.g. https://github.com/rails/rails/blob/master/activesupport/lib/active_support.rb#L28-L30 ).